src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -290,13 +286,9 @@ SaveInProgressIntro.propTypes = {
   lastSavedDate: PropTypes.number,
   user: PropTypes.object.isRequired,
   pageList: PropTypes.array.isRequired,
-  saveInProgress: PropTypes.object.isRequired,
-  fetchInProgressForm: PropTypes.func.isRequired,
-  removeInProgressForm: PropTypes.func.isRequired,
   retentionPeriod: PropTypes.string,
   startText: PropTypes.string,
   pathname: PropTypes.string,
-  toggleLoginModal: PropTypes.func.isRequired,
```

Comment:
I'm curious about the reasoning behind removing all these entries in `propTypes`. I understand they are now set via `mapDispatchToProps`. Maybe I'm just not sure about how we're using `propTypes`. But if it's really just for internally documenting the component it might be better to leave them? Anyhow I'm just looking for some clarity here!

Reply:
I suppose it would be fine to keep for documentation. No harm in keeping them; just haven't been used to declaring types for mapped props.